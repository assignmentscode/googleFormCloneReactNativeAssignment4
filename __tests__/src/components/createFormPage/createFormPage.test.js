import React from 'react';
import renderer from 'react-test-renderer';
import CreateFormPage from '../../../../src/components/createFormPage/createFormPage';

describe('createFormPage', () => {
  it('snapshot should match the previous snapshot', () => {
    const tree = renderer.create(<CreateFormPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
