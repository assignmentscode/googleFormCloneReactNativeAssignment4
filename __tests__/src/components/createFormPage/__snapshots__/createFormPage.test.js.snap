// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createFormPage snapshot should match the previous snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexGrow": 1,
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 80,
        "justifyContent": "center",
        "paddingLeft": 10,
        "paddingRight": 10,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      style={
        Object {
          "borderBottomColor": "gray",
          "borderBottomWidth": 2,
          "flexGrow": 1,
          "textAlign": "center",
          "width": "100%",
        }
      }
      underlineColorAndroid="transparent"
      value="Form Name"
    />
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "height": 50,
          "justifyContent": "center",
          "marginRight": 20,
          "paddingRight": 20,
        }
      }
    >
      <Text
        style={
          Object {
            "backgroundColor": "#3f51b5",
            "color": "black",
            "fontSize": 20,
            "fontWeight": "500",
            "padding": 10,
          }
        }
      >
        ADD FIELD
      </Text>
    </View>
  </View>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    style={
      Object {
        "borderBottomColor": "gray",
        "borderBottomWidth": 2,
        "flexGrow": 1,
        "textAlign": "center",
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
    value="FIELD NAME"
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#3f51b5",
          "height": 60,
          "justifyContent": "center",
        }
      }
    >
      <Text>
        Save
      </Text>
    </View>
  </View>
</View>
`;
